tests/schematics/update/packages/update-package-group-2/package.json

```diff
@@ -0,0 +1,11 @@
+{
+  "name": "@angular-devkit-tests/update-package-group-2",
+  "version": "2.0.0",
+  "description": "Tests",
+  "ng-update": {
+    "packageGroup": {
+      "@angular-devkit-tests/update-package-group-1": "^1",
+      "@angular-devkit-tests/update-package-group-2": ""
```

Comment:
Specifying the owning package again in the group and with an empty version specifier is ambiguous.  From an npm perspective this would mean install "any version" but the user doesn't want "any version".  They want the version of the actual package specified above.  They could put the version there as well but then there are two elements that need to be kept in sync within the same file.

Reply:
Angular does it and it's kind of suggested (since it's just easier to list all packages in all `package.json`). It's supported and it's ignored since it's on the command line.